main.go

```diff
@@ -484,6 +486,16 @@ If you have the Deploy to Bitrise.io step (after this step),
 that will attach the file to your build as an artifact!`, logPath)))
 }
 
+func handleXcprettyInstallError(err error) (string, error) {
+	if IsXcprettyInstallationCheckError(err) {
+		return "", err
+	}
+
+	log.Warnf("%s", err)
```

Comment:
The handleXcprettyInstallError function logging a warning is not really useful, as will fail after if there is an error.

Reply:
The program only fails if the error is an `InstallationCheckError`, otherwise, it just fallbacks to `xcodebuild` and logs a warning message.